leaflet/index.d.ts

```diff
@@ -1360,7 +1360,7 @@ declare namespace L {
         className?: string;
     }
 
-    export interface DivIcon extends Icon {}
+    export class DivIcon extends Icon {}
```

Comment:
Can you add a constructor that takes in a single optional `DivIconOptions`, please.